pkg/watcher/internal/metricsprovider/datadog.go

```diff
@@ -221,11 +221,11 @@ func (s datadogClient) getMetricsHelper(window *watcher.Window, host string) (ma
 	responseContent, _ := json.MarshalIndent(resp, "", "  ")
 	log.Debugf("Response from MetricsApi.QueryTimeseriesData:\n%s\n", responseContent)
 
-	return getMetricsFromTimeSeriesResponse(resp)
+	return GetMetricsFromTimeSeriesResponse(resp)
 }
 
 // This method parses the datadogV2 time series response and return a map with key hostname, value an array of watcher.Metric
-func getMetricsFromTimeSeriesResponse(resp datadogV2.TimeseriesFormulaQueryResponse) (map[string][]watcher.Metric, error) {
+func GetMetricsFromTimeSeriesResponse(resp datadogV2.TimeseriesFormulaQueryResponse) (map[string][]watcher.Metric, error) {
```

Comment:
I don't think we need to make this public as GetMetricsFromTimeSeriesResponse is used within the package.